src/com/cft/contactmerge/MapMaker.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * MapMaker is responsible for creating the map file that tells ContactImporter
+ * and MergeFileExporter how to map properties (FirstName, Address, etc) to
+ * the columns in the file.
+ */
+package com.cft.contactmerge;
+
+import java.util.*;
+
+public class MapMaker {
+    public void addColumnMap(String property, int columnInFile)
+    {
+        // TODO: Implementation needed
+    }
+
+    public void removeColumnMap(String propertyName)
+    {
+        // TODO: Implementation needed
+    }
+
+    public void addStaticMap(String property, String value)
+    {
+        // TODO: Implementation needed
+    }
+
```

Comment:
Maybe adding another method editColumnMap()?

Reply:
There isn't really anything to edit yet though. If you change the value you have defined a new map and so add/remove cover it all. If we add data to the mapping, we will need an edit.